Python/ceval.c

```diff
@@ -689,8 +689,8 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
 #ifdef Py_STATS
     int lastopcode = 0;
 #endif
-    int opcode;        /* Current opcode */
-    int oparg;         /* Current opcode argument, if any */
+    uint8_t opcode;       /* Current opcode */
+    int oparg;            /* Current opcode argument, if any */
```

Comment:
Heh, I would have kept the comments at the same column, so you don't appear to be editing the `oparg` line. :-)

Reply:
Sure -- I think my editor did something magical.